### PR TITLE
New extraction for translatable strings

### PR DESCRIPTION
**Pull request by ptoscano**

### CODE REVIEW

File: po/keys.pot [hunk 1]
```diff
@@ -4179,1122 +4210,1141 @@ msgid "specify an organization"
...
+#: src/subscription_manager/managercli.py:2278
 msgid "No need to update subscriptions at this time."
 msgstr ""
 
-#: src/subscription_manager/managercli.py:2271
-msgid "Attach a specified subscription to the registered system"
+#: src/subscription_manager/managercli.py:2290
+msgid ""
+"Attach a specified subscription to the registered system, when system does "
+"not use Simple Content Access mode"
```
Comment (wottop) [inline po/keys.pot:4719]: when the system does

**Review by wottop — CHANGES_REQUESTED** [review]
Just a grammatical change

File: po/keys.pot (hunk 1 above)
Comment (ptoscano) [reply po/keys.pot:4719]: This is a typo in the code - what do you think about fixing later on?

**Comment by wottop** [comment]
I'm fine with it. It's not a big deal.

**Review by wottop — APPROVED** [review]

**Pull request merged**